env-setup/local_provisioner.tf

```diff
@@ -15,16 +15,16 @@
 
 
 resource "null_resource" "copy_datasets" {
-    provisioner "local-exec" {
-        command = <<EOT
+  provisioner "local-exec" {
+    command = <<EOT
 _REGION=${var.region}
 _PROJECT_ID=${data.google_project.project.project_id}
 _PROJECT_NUMBER=${data.google_project.project.number}
 _NETWORK_ID=${google_compute_network.network.id}
 _NETWORK_ID=$(echo $_NETWORK_ID | sed "s#$_PROJECT_ID#$_PROJECT_NUMBER#")
 _FILESTORE_IP=${google_filestore_instance.filestore_instance.networks.0.ip_addresses.0}
 _FILESHARE=${google_filestore_instance.filestore_instance.file_shares.0.name}
-_GCS_PATH=${var.gcs_dbs_path}
+_GCS_PATH=gs://${var.gcs_dbs_path}
```

Comment:
@chmstimoteo Could you please change the README file explaining how the user must format this GCS path?  
This is the current explanation: 

```
GCS_DBS_PATH - the path to the GCS location of the genetic databases and model parameters. Terraform will copy the databases replicating a folder structure on GCS. Terrafom will also copy model parameters to the regional bucket. The parameters should be in the <GCS_DBS_PATH>/params
```

Reply:
Fixed. Pls check.